netdata-installer.sh

```diff
@@ -44,15 +44,21 @@ else
 	source "${NETDATA_SOURCE_DIR}/packaging/installer/functions.sh" || exit 1
 fi
 
-download() {
+download_go() {
 	url="${1}"
 	dest="${2}"
+
 	if command -v curl >/dev/null 2>&1; then
-		run curl -sSL --connect-timeout 10 --retry 3 "${url}" >"${dest}" || fatal "Cannot download ${url}"
+		run curl -sSL --connect-timeout 10 --retry 3 "${url}" >"${dest}"
 	elif command -v wget >/dev/null 2>&1; then
-		run wget -T 15 -O - "${url}" >"${dest}" || fatal "Cannot download ${url}"
+		run wget --waitretry 3 -T 10 -O - "${url}" >"${dest}"
```

Comment:
There was a reason we did the timeout and retries for wget like this. Please put it back as it was, we had tested thoroughly this part of the code.

Reply:
The idea was to align the way we retry and timeout on curl, on wget.
I am okay with any values, as long as we use the same pattern for either curl or wget so that we use consistent settings between the two